autoemulate/core/compare.py

```diff
@@ -72,8 +79,8 @@ def __init__(  # noqa: PLR0913
             Defaults to None, in which case the data is standardized.
         model_tuning: bool
             Whether to tune the hyperparameters of the models using cross-validation.
-            If False, the models use the model_kwargs provided. Defaults to True.
-        model_kwargs: ModelParams | None
+            If False, the models use the model_params provided. Defaults to True.
+        model_params: ModelParams | None
```

Comment:
Add `transformed_emulator_params` to docstring

Reply:
Added.